examples/02-Maxwell/Maxwell3DTeam7.py

```diff
@@ -0,0 +1,380 @@
+        48.25,
+        51.35,
+        53.35,
+        45.37,
+        24.01,
+    ],
+    [-1.35, -0.71, -0.81, -0.67, 0.15, 1.39, 2.67, 3.00, 4.01, 3.80, 4.00, 3.02, 2.20, 2.78, 1.58, 1.37, 0.93],
+]
+
+# Dataset detail is used to encode test parameters in the text files.
+# For example 'Bz A1_B1 050 0' is the z component of Flux Density, B along line A1_B1 at 50Hz and 0deg
+# These text files are created and saved in the default project_dir
+print("project_dir", project_dir)
+dataset_range = range(int(0), len(dataset), int(1))
+line_length_range = range(int(0), len(line_length), int(1))
+dataset_list = []
+
+for item in dataset_range:
+    dataset_list.clear()
+    for jtem in line_length_range:
+        dataset_list.insert(jtem, data[item][jtem])
+        ziplist = zip(line_length, dataset_list)
+    with open(project_dir + "\\" + str(dataset[item]) + ".csv", "w", newline="") as f:
+        writer = csv.writer(f, delimiter=",")
+        writer.writerow(header)
+        writer.writerows(ziplist)
+
+# Create rectangular plots and use text file encoding to control formatting of the rectangular plots
+# DC plot is created separately, it needs a different frequency and phase to the other plots.
+for item in dataset_range:
+    if item % 2 == 0:
+        plotname = dataset[item][0:3] + "Along the Line" + dataset[item][2:9] + ", " + dataset[item][9:12] + "Hz"
+        if dataset[item][9:12] == "000":
+            Plot.CreateReport(
+                plotname,
+                "Fields",
+                "Rectangular Plot",
+                "Setup1 : LastAdaptive",
+                ["Context:=", "Line_" + dataset[item][3:8], "PointCount:=", 1001],
+                [
+                    "Distance:=",
+                    ["All"],
+                    "Freq:=",
+                    [str(dc_freq) + "Hz"],
+                    "Phase:=",
+                    ["0deg"],
+                    "Coil_Excitation:=",
+                    ["All"],
+                ],
+                ["X Component:=", "Distance", "Y Component:=", [dataset[item][0:2]]],
+            )
+        else:
+            Plot.CreateReport(
+                plotname,
+                "Fields",
+                "Rectangular Plot",
+                "Setup1 : LastAdaptive",
+                ["Context:=", "Line_" + dataset[item][3:8], "PointCount:=", 1001],
+                [
+                    "Distance:=",
+                    ["All"],
+                    "Freq:=",
+                    [dataset[item][9:12] + "Hz"],
+                    "Phase:=",
+                    ["0deg", "90deg"],
+                    "Coil_Excitation:=",
+                    ["All"],
+                ],
+                ["X Component:=", "Distance", "Y Component:=", [dataset[item][0:2]]],
+            )
+
+        # Import test data into the correct plot and overlay it with simulation results.
+        if item == 0:
+            Plot.ImportIntoReport(plotname, project_dir + "\\" + str(dataset[item]) + ".csv")
```

Comment:
If possible, I would just suggest to use the os.path.join here instead of the string concatenation. Same for the following lines.
```suggestion
            Plot.ImportIntoReport(plotname, os.path.join(project_dir, str(dataset[item]) + ".csv"))
```

Reply:
I'll commit this change, but just wanted to understand why this was better